frontend/src/components/form/api/SuccessIcon.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <transition name="fade">
+    <div
+      v-if="visible"
+      class="out-of-flow-parent">
+      <v-avatar
+        color="white"
+        class="out-of-flow-child"
+        size="32">
+        <v-icon
+          color="green"
+          size="24">
+          mdi-check
+        </v-icon>
+      </v-avatar>
+    </div>
+  </transition>
+</template>
+
+<script>
+export default {
+  name: 'SuccessIcon',
+  props: {
+    visible: {
+      type: Boolean,
+      default: true
+    }
+  }
+}
+</script>
+
+<style scoped>
+.fade-enter-active, .fade-leave-active {
+  transition: opacity 1s;
+}
```

Comment:
Feedback(enter) sollte schneller da sein, als das leave. Ausserdem kann das leave unaufälliger sein.

```suggestion
.fade-enter-active {
  transition: opacity 250ms;
}

.fade-leave-active {
  transition: opacity 5s ease-in;
}
```